### PR TITLE
Personal Portfolio Works And About Sections

### PR DESCRIPTION
**Pull request by shaqdeff**
- Created a works and about section using HTML and CSS.
- For the works section, I used CSS Grid and FlexBox as per the requirements.
- As for the about section, 'ul li' were used for the languages categories.
- Set up and checked linters.
- Used best HTML and CSS practices.
- Used proper GitHub flow.

Kindly review and provide feedback if any changes need to be made.

Regards.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -67,5 +68,180 @@ <h1>
...
+          </ul>
+          <button class="btn">See Project</button>
+        </div>
+      </section>
+    </section>
+    <section class="about-section">
+      <h1>
+        About <br />
+        Myself
+      </h1>
```
Comment (Wes-Isaac) [inline index.html]: - [ ] Great job so far, it's a good practice to have one `h1` in your page, please change this to `<hx>(<h2><h3>..)`

File: index.html [hunk 2]
```diff
@@ -67,5 +68,180 @@ <h1>
...
+      <p>
+        I’m a software developer! <br />
+        I can help you build a product, feature or website. Look through some of
+        my work and experience! If you like what you see and
+        <br />
+        have a project you need coded, <br />
+        don’t hestiate to contact me.
+      </p>
+      <p>LET'S CONNECT</p>
```
Comment (Wes-Isaac) [inline index.html:199]: - [ ] Great job, it's one of the requirements to use `<hx>(<h2><h3>..)` for this. Kindly fix it.

**Review by Wes-Isaac — CHANGES_REQUESTED** [review]
Hi @shaqdeff, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Linters are on point ✔
- Splendid work with the README file 👀
- Proper use of GitHub flow 👌
- Awesome design 🙌


## Required Changes ♻️
_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by Mre55 — APPROVED** [review]
**Hi @shaqdeff,**

# Status: APPROVED 🎉 🎊 

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚀: 
Congratulations! 🎉 

Alt Text

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/styles.css [hunk 3]
```diff
@@ -131,7 +137,47 @@ nav {
...
   display: grid;
-  grid-template: repeat(4, 20rem) / repeat(1, 30rem);
+  grid-template-columns: 1fr;
   min-height: 100vh;
   margin-top: 10rem;
+  justify-items: center;
+}
+
+.card {
+  /* align-items: center; */
```
Comment (jodrack07) [inline css/styles.css]: Please, do not leave unused code as comments in your codebase to make your codebase cleaner.

File: index.html [hunk 4]
```diff
@@ -67,13 +67,91 @@ <h1>
...
+            sign-ups required.
+          </p>
+          <ul class="categories">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+          <button class="btn">See Project</button>
+        </div>
+      </section>
```
Comment (jodrack07) [inline index.html:98]: Using ```<section>``` inside ```<section>``` is not a good practice. Consider using ```<article>``` instead.
Like ```<section>
               <arcticle></article>
          </section>```

**Review by jodrack07 — COMMENTED** [review]
Try thinking and implementing the suggestions

**Pull request merged**